### ISSUE TITLE
None of the custom renderers change mouse cursor

### ISSUE THREAD

**Issue by denballakh** [pyimgui/pyimgui#365]
Imgui wants to change cursor when it is hovering over something reactive (window edge for resizing, for example).
pyimgui lacks this functionality and cursor is always the default one which feels counterintuitive.

I'm using windows 10.
I've tested pygame renderer.

**Comment by Aman-Anas** [comment]
Yeah, I noticed this issue as well with the glfw renderer. It's not too difficult to implement a software rendered cursor (probably not as fast as a hardware cursor but more than good enough for my use cases) with your own cursor images loaded from file, and drawn on top using the imgui foreground draw list. Would be good to have a builtin solution though.

**Comment by learn-more** [comment]
`pyglet` changes the cursor for me:
![image](https://github.com/pyimgui/pyimgui/assets/610685/b2ec74aa-853a-425e-8dc3-bd25f053e204)

**Comment by learn-more** [comment]
@KinoxKlark Against which branch should we make PR's for this?

**Comment by denballakh** [comment]
pyglet indeed has this feature, i can confirm
it also has this feature: when imgui viewport is not focused and you interact with it in any way (hover over something, scroll something, ...), it responds.
pygame and glfw backends process events only if their window is focused

it also works well with my gpu, i no longer see artifacts (they are not caused by imgui or pyimgui, i guess they are caused by bad gpu drivers)
example of such artifact:
![image](https://github.com/pyimgui/pyimgui/assets/47365157/950acc7f-3f30-479c-b45c-fd57cc5fef87)
